Library/Homebrew/extend/os/dependency_collector.rb

```diff
@@ -1,2 +1,3 @@
 require "dependency_collector"
 require "extend/os/mac/dependency_collector" if OS.mac?
+require "extend/os/linux/dependency_collector" if OS.linux?
```

Comment:
Remove this line.

Reply:
had to add it back to override `xz_dep_if_needed`